.github/workflows/containerized-tests.yaml

```diff
@@ -0,0 +1,126 @@
+name: Integration and E2E Testing
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+    paths:
+      - "app/**"
+      - "api/**"
+      - ".github/workflows/containerized-tests.yaml"
+  workflow_dispatch:
+
+# ENV values match local development seed values only.
+# Does not allow for Keycloak-Express or GC Notify tests
+env:
+  TESTING: true
+  API_PORT: 3004
+  MONGO_PORT: 27017
+  MONGO_PASSWORD: purchase-admin
+  MONGO_USERNAME: purchase-admin
+  MONGO_DATABASE: purchase-db
+  MONGO_SERVICE: mongo
+  ENVIRONMENT: local
+  FRONTEND_PORT: 8080
+  SSO_CLIENT_ID: citz-imb-sandbox-4796
+  SSO_CLIENT_SECRET: ${{ secrets.SSO_CLIENT_SECRET }}
+  SSO_AUTH_SERVER_URL: https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect
+  TEST_USERNAME: ${{ secrets.TEST_USERNAME }}
+  TEST_PASSWORD: ${{ secrets.TEST_PASSWORD }}
+
+jobs:
+  containerized-tests:
+    name: Containerized Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      # Ups the rate limiter max to an unreasonable number. Avoids 429 responses.
+      - name: Relax Express Rate Limiter
```

Comment:
This obviously works but it might be more elegant to change the values within express.js based on when the TESTING variable is true

Reply:
Turns out I had already tried to account for this and just forgot about it. 
BUT looks like my logic was backwards, so this is a good catch.
<img width="323" alt="image" src="https://github.com/bcgov/citz-imb-staff-purchasing-reimbursement/assets/37922247/e9121324-9404-407e-8c55-b1c0200da748">